baremetalpxeserver/ansible/baremetalpxeserver.py

```
@@ -125,11 +118,23 @@
 else:
     error("unsupported OS!")
 
+# name: install virtualenv
+py_version = get_virtualenv_python_version(virtenv_path, host_post_info)
```

Ruff (0.14.7)

`get_virtualenv_python_version` may be undefined, or defined from star imports

(F405)